internal/api/initialize/init_resources_test.go

```diff
@@ -0,0 +1,60 @@
+package initialize
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	obs "github.com/openshift/cluster-logging-operator/api/observability/v1"
+	"github.com/openshift/cluster-logging-operator/internal/utils"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var _ = Describe("#Resources", func() {
+	var (
+		forwarder    obs.ClusterLogForwarder
+		expResources = &v1.ResourceRequirements{
+			Limits: v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("1024Mi"),
+				v1.ResourceCPU:    resource.MustParse("6000m"),
+			},
+			Requests: v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("64Mi"),
+				v1.ResourceCPU:    resource.MustParse("500m"),
+			},
+		}
+	)
+
+	BeforeEach(func() {
+		forwarder = obs.ClusterLogForwarder{
+			Spec: obs.ClusterLogForwarderSpec{},
+		}
+	})
+
+	It("should apply the default resources when collector is not defined", func() {
+		forwarder := Resources(forwarder, utils.NoOptions)
+		Expect(forwarder.Spec.Collector.Resources).To(Equal(expResources))
+	})
+	It("should apply the default resources when resources are not defined defined", func() {
```

Comment:
nit: You wrote defined twice
```suggestion
	It("should apply the default resources when resources are not defined", func() {
```